README.md

```diff
@@ -83,19 +83,23 @@ See also Troubleshooting.
 
 -  Run this:
 
-      `github_changelog_generator -u github_username -p github_project`
-      `github_changelog_generator  github_username/github_project`
+	   $ github_changelog_generator -u github_username -p github_project
+	   
+or
+
+	   $ github_changelog_generator github_username/github_project
+	   
 
 - For Github Enterprise repos, specify *both* `--github-site` and `--github-api` options:
 
-      github_changelog_generator --github-site="https://github.yoursite.com" \
-                                 --github-api="https://github.yoursite.com/api/v3/"
+       $ github_changelog_generator --github-site="https://github.yoursite.com" \
+                                  --github-api="https://github.yoursite.com/api/v3/"
 
 This generates a `CHANGELOG.md`, with pretty Markdown formatting.
 
 ### Params
 
-Type `github_changelog_generator --help` for details.
+Type `$ github_changelog_generator --help` for details.
```

Comment:
Here, to avoid confusing readers use an indented section.

```
Print help for all command-line options to learn more details:

    $ github_changelog_generator --help
```